modules/virt-understanding-live-migration.adoc

```diff
@@ -17,3 +17,8 @@ which it is running is placed into maintenance.
 Virtual machines must have a persistent volume claim (PVC)
 with a shared ReadWriteMany (RWX) access mode to be live migrated.
 ====
+
```

Comment:
My only observation is that it's a little weird to have two notes in a row, and I can't tell if the first note really needs to be designated as more important than the second note.